_includes/start_in_docker/mac-linux-steps.md

```diff
@@ -0,0 +1,160 @@
+- `-d`: This flag runs the container in the background so you can continue the next steps in the same shell. 
+- `--name`: The name for the container. This is optional, but a custom name makes it significantly easier to reference the container in other commands, for example, when opening a Bash session in the container or stopping the container. 
+- `--hostname`: The hostname for the container. You will use this to join other containers/nodes to the cluster.
+- `--net`: The bridge network for the container to join. See step 1 for more details.
+- `-p 26257:26257 -p 8080:8080`: These flags map the default port for inter-node and client-node communication (`26257`) and the default port of HTTP requests from the Admin UI (`8080`) from the container to the host. This enables inter-container communication and makes it possible to call up the Admin UI from a browser.
+- `-v "${PWD}/cockroach-data/roach1:/cockroach/cockroach-data"`: This flag mounts a host directory as a data volume. This means that data and logs for this node will be stored in `${PWD}/cockroach-data/roach1` on the host and will persist after the container is stopped or deleted. For more details, see Docker's <a href="https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-host-directory-as-a-data-volume">Mount a host directory as a data volume</a> topic.
+- `cockroachdb/cockroach:{{site.data.strings.version}} start --insecure`: The CockroachDB command to [start a node](start-a-node.html) in the container in insecure mode. 
+
+  {{site.data.alerts.callout_success}}By default, each node's cache is limited to 25% of available memory. This default is reasonable when running one container/node per host. When running multiple containers/nodes on a single host, however, it may lead to out of memory errors, especially when testing against the cluster in a serious way. To avoid such errors, you can manually limit each node's cache size by setting the <a href="start-a-node.html#flags"><code>--cache</code></a> flag in the <code>start</code> command.{{site.data.alerts.end}}
+
+## Step 3. Start additional containers/nodes
+
+~~~ shell
+# Start the second container/node:
+$ docker run -d \
+--name=roach2 \
+--hostname=roach2 \
+--net=roachnet \
+-P \
+-v "${PWD}/cockroach-data/roach2:/cockroach/cockroach-data" \
+cockroachdb/cockroach:{{site.data.strings.version}} start --insecure --join=roach1
+
+# Start the third container/node:
+$ docker run -d \
+--name=roach3 \
+--hostname=roach3 \
+--net=roachnet \
+-P \
+-v "${PWD}/cockroach-data/roach3:/cockroach/cockroach-data" \
+cockroachdb/cockroach:{{site.data.strings.version}} start --insecure --join=roach1
+~~~
+
+These commands add two more containers and start CockroachDB nodes inside them, joining them to the first node. There are only a few differences to note from step 2:
+
+- `-P`: This flag maps exposed ports to random ports on the host. This random mapping is fine since we've already mapped the relevant ports for the first container.
```

Comment:
Why are we mapping these ports to the host at all? I can see the argument for mapping roach1's ports to enable easy access to the admin UI at a known location, but what does mapping these two containers' ports buy us?